config/routes.rb

```diff
@@ -6,5 +6,6 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+
+  root to: "messages#show"
 end
```

Comment:
- [ ] Kindly check your  root route properly. It is supposed to point to the root#index. Something like this. 

``` 
  root "root#index"
```
and there should be a route for that view.  At this moment, you don't have index for your view. This is what you have :point_down: 

![view-show](https://github.com/Abdulwhbhussain/hello-rails-react/assets/26597432/1126c184-5cba-416a-a46e-055e042d4116)

### Your page after starting the app.

![result](https://github.com/Abdulwhbhussain/hello-rails-react/assets/26597432/98d8f206-7caa-44a9-a4b7-3a1cdfa0e0d8)


- [ ] There should a route to generate random greetings in your route.rb. That, is not implemented at this moment. Kindly do that for your app to work as expected. Kindly look at bellow snippet to gain some insight


```
   resources :greetings, only: [:index], defaults: {format: 'json'} do
    collection do
      get :random_greeting
    end
  end  

```

Reply:
Sure! I fixed it!